Python/bltinmodule.c

```diff
@@ -3014,9 +3014,16 @@ static PyMethodDef builtin_methods[] = {
 };
 
 PyDoc_STRVAR(builtin_doc,
-"Built-in functions, exceptions, and other objects.\n\
+"Built-in functions, types, exceptions, and other objects.\n\
 \n\
-Noteworthy: None is the `nil' object; Ellipsis represents `...' in slices.");
+This module provides direct access to all 'built-in'\n\
+identifiers of Python; for example, builtins.open is\n\
+the full name for the built-in function open().\n\
```

Comment:
```suggestion
identifiers of Python; for example, builtins.len is\n\
the full name for the built-in function len().\n\
```
`open()` is a slightly bad example because it thinks it's in the `io` module instead:

```
>>> import builtins
>>> builtins.open.__module__
'io'
```